core/src/main/scala/kafka/server/AlterIsrChannelManager.scala

```diff
@@ -0,0 +1,121 @@
+package kafka.server
```

Comment:
It would be good to find a better name for this.  When I read "AlterIsrChannelManager" I assumed it had its own separate channel, rather than using the ControllerChannelManager.

Reply:
Yea, maybe just "AlterIsrManager"?